test/services/database-analyzer/database-analyzer-sequelize.js

```diff
@@ -35,6 +36,31 @@ describe('Database analyser > Sequelize', () => {
         await sequelizeHelper.close();
       });
 
+      it('should give us constraint_name, table_name_ column_type, column_name, foreign_table_name, foreign_column_name, unique_indexes', async () => {
```

Comment:
A clearer french sentence should be more explicit: should provide the schema elements (for example, create your own)